docs/notebook/Tutorial.ipynb

```diff
@@ -726,7 +733,7 @@
    ],
    "source": [
     "conf = OmegaConf.load('../source/env_interpolation.yaml')\n",
-    "print(conf.pretty(resolve=True))"
```

Comment:
This example was making a point of showing the effect of resolve.
you removed resolve, but now it's not making any point because resolve is True by default.

Reply:
Changed in the last commit, now the next cell shows wihout resolving.